CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Fixed
+
+* [#465](https://github.com/NVIDIA/NeMo-Guardrails/pull/465) Fix empty response for multiple messages  by @saradiazdelser.
```

Comment:
Using `-` for unordered lists is a valid syntax. You should use the existing style rather than change the whole file.